animals/routes.js

```diff
@@ -0,0 +1,12 @@
+const routes = require("express").Router();
+
+const restrict = require("../auth/restrict.js");
+const Animals = require("./model.js");
+
+routes.get("/", restrict, async (req, res) => {
```

Comment:
good job creating your get endpoint in it's own route and restricting it. way to go above and beyond!